### PR TITLE
Applied changes for Console v0.24.0

### LINKED ISSUE #747: [RELEASE] Console v0.24.0
Console [v0.24.0](https://github.com/minio/console/releases/tag/v0.24.0) has changes that impact the docs:

- [x] Enable/disable Call Home [PR #2248](https://github.com/minio/console/pull/2248)
- [x] Tooltips show environment variable for settings [PR #2636](https://github.com/minio/console/pull/2636)
- [x] Export/Import server configuration settings [PR #2664](https://github.com/minio/console/pull/2664)
- [x] Health component removed [PR #2659](https://github.com/minio/console/pull/2659)
- [x] Object Manager button restored [2665](https://github.com/minio/console/pull/2665)

### PR DESCRIPTION
**Pull request by djwfyi**
- Adds information about call home to the console subnet registration page
- Mentions viewing environment variables with the tooltip
- Adds info about excporitng or importing settings
- Adds object manager button to the managing objects page

Closes #747

### CODE REVIEW

**Review by feorlen — APPROVED** [review]
LGTM

File: source/administration/console/managing-objects.rst [hunk 1]
```diff
@@ -47,6 +48,11 @@ Example actions the user may be able to perform include:
 - Display versions
 - Delete
 
+.. versionadded:: Console v0.24.0
+
+   View the status of uploading or downloading objects with the object manager button available on the top right corner of many Console screens.
```
Comment (ravindk89) [inline source/administration/console/managing-objects.rst:53]: I'd maybe replace "many" with "select" - "many" implies a certain % of screens, which we don't reasonably know. I'm actually going to guess you can probably see it only on the object browser view and subviews, but that's a quibble.

**Review by ravindk89 — APPROVED** [review]
LGTM mod nits

File: source/administration/console/managing-objects.rst (hunk 1 above)
Comment (djwfyi) [reply source/administration/console/managing-objects.rst:53]: Actually, you can see it most everywhere.

File: source/administration/console/managing-deployment.rst [hunk 2]
```diff
@@ -151,6 +152,11 @@ This section contains the following subsections.
 - Logger Webhook
 - Audit Webhook
 
+.. versionadded:: Console v0.24.0
+
+   Environment variable configuration settings override any customizations added in the MinIO Console.
+   When environment variables control an option, hover the mouse cursor over a configuration field to show a tooltip message that specifies which environment variable overrides the configuration.
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/administration/console/managing-deployment.rst:158]: Taking your suggestion with a slight tweak.

**Pull request merged**